Project/Source Files/Services/Networking/Helper.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  Helper.swift
+//  NetguruCollegeApp
+//
+
+import Foundation
+struct Helper {
+    func convertToOneString(array: [String], with separator: String) -> String {
+        return array.joined(separator: separator)
```

Comment:
Actually, this method could be used just like that in this case in my opinion, so this method won't be needed :) But it's up to you.